### DISCUSSION TITLE
What is the minimum zsh version supported?

### DISCUSSION THREAD

**Discussion by wookayin** [mattmc3/antidote#76] [Q&A] [answered]
I found that antidote doesn't work on old zsh versions, say 5.1 (which is Ubuntu 16.04). I know Ubuntu 16.04 is EOL and zsh 5.1 is such a legacy version, but I had some unusual rare cases where I had to run antidote (and my dotfiles) on such legacy systems.

We would expect users to use some modern versions of zsh, but all that said, what would be the minimum zsh version (officially) supported? It'd be also a good idea to document this if applicable. Thanks!

**Comment by mattmc3** [comment, marked as answer]
5.4.2 is the oldest supported version, [which was released in 2017](https://zsh.sourceforge.io/News/) and is tested via CI in a GitHub workflow on Ubuntu 18.04 LTS.

https://github.com/mattmc3/antidote/blob/8257445d3235dd70c072712cf0b00fa1723435aa/.github/workflows/test.yml#L31
